app/src/main.ts

```diff
@@ -29,6 +29,11 @@ if (require('electron-squirrel-startup')) {
 
 const appArgs = JSON.parse(fs.readFileSync(APP_ARGS_FILE_PATH, 'utf8'));
 
+// Take in a URL on the command line as an override
+if (process.argv.length > 1) {
+  appArgs.targetUrl = process.argv[1];
```

Comment:
Sure, but please validate that `process.argv[1]` is a URL.

```suggestion
  const maybeUrl = process.argv[1];
  try {
    new URL(maybeUrl);
    appArgs.targetUrl = maybeUrl;
  } catch (err) {
    // Not doing anything
  }
```